protocol/app/upgrades/v3.0.0/upgrade.go

```diff
@@ -93,6 +102,52 @@ func InitializeModuleAccs(ctx sdk.Context, ak authkeeper.AccountKeeper) {
 	}
 }
 
+func IcaHostKeeperUpgradeHandler(
+	ctx sdk.Context,
+	vm module.VersionMap,
+	mm *module.Manager,
+) {
+	icaAppModule, ok := mm.Modules[icatypes.ModuleName].(ica.AppModule)
+	if !ok {
+		panic("Modules[icatypes.ModuleName] is not of type ica.AppModule")
+	}
+	// Add Interchain Accounts host module
+	// set the ICS27 consensus version so InitGenesis is not run
+	// initialize ICS27 module
+	vm[icatypes.ModuleName] = icaAppModule.ConsensusVersion()
+
+	// controller submodule is not enabled.
+	controllerParams := icacontrollertypes.Params{}
+
+	// host submodule params
+	hostParams := icahosttypes.Params{
+		HostEnabled: true,
+		AllowMessages: []string{
+			// IBC transfer messages
+			sdk.MsgTypeURL(&ibctransfertypes.MsgTransfer{}),
+
+			// Bank messages
+			sdk.MsgTypeURL(&banktypes.MsgSend{}),
+
+			// Staking messages
+			sdk.MsgTypeURL(&stakingtypes.MsgDelegate{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgBeginRedelegate{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgUndelegate{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgCancelUnbondingDelegation{}),
+
+			// Distribution messages
+			sdk.MsgTypeURL(&distrtypes.MsgSetWithdrawAddress{}),
```

Comment:
Would it be possible to add `"/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward"` here? 

Stride uses it [here](https://github.com/Stride-Labs/stride/blob/8c2dcb993c458b8a2244b0acbcfe61eb74075270/x/stakeibc/keeper/msg_server_submit_tx.go#L190) to withdraw accrued staking rewards

Reply:
Done - accidentally put `MsgWithdrawValidatorCommission` which is not needed IIUC instead of `MsgWithdrawDelegatorReward`